sdk/trace/src/test/scala/org/typelevel/otel4s/sdk/trace/SdkSpanBackendSuite.scala

```diff
@@ -218,23 +221,15 @@ class SdkSpanBackendSuite extends CatsEffectSuite with ScalaCheckEffectSuite {
         )
 
         _ <- assertIO(
-          extra.view
-            .map(_.key)
-            .filterNot(init.contains)
-            .toList
-            .traverse(span.getAttribute(_)),
+          extra.toList.traverse(a => span.getAttribute(a.key)),
           List.fill(extra.size)(None)
```

Comment:
The tests kept failing. Even though the keys are filtered, the 'expected' number of attributes depends on the non-filtered size.